React/wbooks/src/app/screens/Home/components/Book/index.js

```diff
@@ -21,7 +25,12 @@ const Book = props => (
 );
 
 Book.propTypes = {
-  book: PropTypes.object.isRequired
+  book: PropTypes.shape({
+    id: PropTypes.number.isRequired,
+    image_url: PropTypes.string,
+    title: PropTypes.string.isRequired,
+    author: PropTypes.string.isRequired
+  })
```

Comment:
Extract books proptypes in a separated file and reuse it y each component what are u using books